utils.js

```diff
@@ -0,0 +1,68 @@
+// TODO
+// await use axios post seedData fn to push data to db
+// done: delete file directory(or add to gitignore so zip files doesn’t push to Github) + drain
+
+const fs = require('fs');
+const path = require('path');
+const unzipper = require('unzipper'); // npm lib used to unzip files
+const { createReadStream } = require('fs');
+
+// Async fs reads and unzip files to a directory
+const folderName = 'Archive.zip';
+const pathToExtract = 'unzippedArchive';
+
+const unzipFolder = (zippedFolder, destination) => {
+  fs.createReadStream(`./${zippedFolder}`).pipe(
+    unzipper.Extract({ path: __dirname + `/${destination}` })
+  );
+};
+
+class File {
```

Comment:
I don't think this needs to be a class, and usually a class is in a file by itself and not surrounded by util functions.

Reply:
What would be a good use case for a class? A file wouldn't be a good use case for a class? There can be many actions or methods to a file, read it, parse it, format it, send it somewhere else.

Reply:
I would defined a `class` as a template for a collection of methods and properties that are related. 

The reason why I don't think this should be a class is because you only have one real method in it. You may add more methods at a later time then I think we should refactor it into a class when that time comes. If you're 100% sure you're adding more methods in the near future then add a `TODO` so other people on your team know what you will do.

Reply:
Can you also change the name of the `readFile` method into something more descriptive of what it's doing, like `getFileContent`.

Reply:
I understand what you are saying... I'll leave todo's for now on!